examples/text_to_image/train_text_to_image_lora.py

```diff
@@ -916,7 +1024,7 @@ def collate_fn(examples):
     pipeline = pipeline.to(accelerator.device)
 
     # load attention processors
-    pipeline.unet.load_attn_procs(args.output_dir)
+    pipeline.load_lora_weights(args.output_dir, weight_name="pytorch_lora_weights.bin")
```

Comment:
Do we have to specify `weight_name`?

Reply:
We don't need to specify this attribute actually. Could you upgrade your local installation of `safetensors`?

Reply:
It works. Thank you! I fixed it.